Gemfile

```diff
@@ -25,6 +25,8 @@ gem "jsbundling-rails"
 gem "jsonb_accessor"
 gem "mail-notify"
 gem "okcomputer"
+gem "omniauth_openid_connect"
+gem "omniauth-rails_csrf_protection"
```

Comment:
[This gem](https://github.com/cookpad/omniauth-rails_csrf_protection) says it mitigates [CVE-2015-9284](https://nvd.nist.gov/vuln/detail/CVE-2015-9284)

`CVE-2015-9284` affects the Ruby `omniauth` gem until version 2.0.0 excluding, but we're using 2.1.2

Do we actually need it? 🤔

Reply:
iirc I added it because something wasn't working without it. Don't remember what, though, I'll try removing it.

Reply:
👍